pymongo/common.py

```diff
@@ -406,7 +406,7 @@ def validate_read_preference_tags(name, value):
             tags = {}
             for tag in tag_set.split(","):
                 key, val = tag.split(":")
-                tags[unquote_plus(key)] = unquote_plus(val)
+                tags[unquote(key)] = unquote(val)
```

Comment:
Oh these are now breaking changes for readPreferenceTags and authMechanismProperties. Please revert this and open a new PYTHON ticket.

Reply:
Done.